test/intl402/RelativeTimeFormat/constructor/constructor/options-toobject-prototype.js

```diff
@@ -0,0 +1,35 @@
+// Copyright 2018 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-Intl.RelativeTimeFormat
```

Comment:
a reference to the [`resolvedoptions`](http://tc39.github.io/proposal-intl-relative-time/#sec-intl.relativetimeformat.prototype.resolvedoptions) method would help a lot here or at least anything in the info part.